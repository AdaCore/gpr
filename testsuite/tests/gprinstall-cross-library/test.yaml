description:
    Check for cross library  installation
driver: python_script
control:
- [SKIP, "'windows' not in env.host.platform", "DEAD on not windows"]
