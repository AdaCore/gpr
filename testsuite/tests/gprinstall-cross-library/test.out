cp /files/src/prj.ads /inst/include/prj/prj.ads

cp /files/lib/prj.ali /inst/lib/prj/prj.ali

cp /files/lib/libprj.so /inst/lib/prj/libprj.so

cp /inst/lib/prj/libprj.so /inst/lib/libprj.so

